api/internal/core/entity/format.go

```diff
@@ -17,26 +17,30 @@
 package entity
 
 import (
-	"log"
 	"strconv"
 	"strings"
+
+	"github.com/apisix/manager-api/log"
 )
 
 func NodesFormat(obj interface{}) interface{} {
 	var nodes []*Node
-	if value, ok := obj.(map[string]float64); ok {
+	switch objType := obj.(type) {
+	case map[string]float64:
+		log.Infof("nodes type: %v", objType)
 		var strArr []string
+		value := obj.(map[string]float64)
 		for key, val := range value {
 			node := &Node{}
 			strArr = strings.Split(key, ":")
 			if len(strArr) != 2 {
-				log.Println("length of string array is not 2")
+				log.Warn("length of string array is not 2")
 				return obj
 			}
 
 			port, err := strconv.Atoi(strArr[1])
 			if err != nil {
-				log.Println("parse int fail:", err)
+				log.Errorf("parse int fail:", err)
```

Comment:
Missing format prompt like `%v` or `%s`, the `%s` is preferred since we don't want to see the dynamic type of error, we only need the literal string. This lint error can be detected by golangci-lint.

Reply:
fixed.